api/controllers/EAppEligibilityQuestionsController.js

```diff
@@ -2,11 +2,11 @@ const HelperService = require("../services/HelperService");
 const EAppEligibilityQuestionsController = {
     renderEligibilityQuestion(req, res) {
         const eligibilityViews = {
-            'apostille-accepted-in-destination':
+            'check-documents-are-eligible':
                 'eApostilles/eligibilityQuestionOne.ejs',
-            'documents-eligible-for-service':
+            'check-recipient-accepts-eapostilles':
                 'eApostilles/eligibilityQuestionTwo.ejs',
-            'pdfs-digitally-signed': 'eApostilles/eligibilityQuestionThree.ejs',
+            'check-documents-are-prepared': 'eApostilles/eligibilityQuestionThree.ejs',
         };
```

Comment:
I like this view naming change and subsequent ones! Immediately more readable and coherent for me, someone not very familiar with the project.